modules/dnn/perf/perf_gemm.cpp

```diff
@@ -0,0 +1,148 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "perf_precomp.hpp"
+#include <opencv2/dnn/shape_utils.hpp>
+
+namespace opencv_test {
+
+struct GemmParam_t {
+    int shapeIn0[2]; // Mat A
+    int shapeIn1[2]; // Mat B
+    double declared_flops;
```

Comment:
flops should be calculated based on the input shape.

Reply:
Thanks for the code reviewing. This part code is referred from 'https://github.com/opencv/opencv/blob/4.x/modules/dnn/perf/perf_convolution.cpp#L26'.

Reply:
We should move forward. How do I know if the given flops is correct or not if there is no calculation?